pyproject.toml

```diff
@@ -12,6 +12,7 @@ dependencies = [
     "sqlmodel>=0.0.14",
     "tidb-vector>=0.0.14",
     "pymysql>=1.1.1",
+    "aiohttp==3.11.18",
```

Comment:
aiohttp is a dependency introduced by the optional dependency litellm, and we shouldn't fix its version in the required dependencies. Maybe we just need to update the uv.lock file

https://github.com/BerriAI/litellm/blob/d408814978b2064c44f65cf1e02c169360c32785/pyproject.toml#L31

Reply:
yes also found that, will take a try

Reply:
addressed